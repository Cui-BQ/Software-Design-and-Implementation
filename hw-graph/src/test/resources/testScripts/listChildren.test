# List children for bides that have 0, 1 and 2 children node.

# Create graphs
CreateGraph graph

# Add 3 node to graph
AddNode graph n0
AddNode graph n1
AddNode graph n2

# Add one child for n1
AddEdge graph n1 n2 e2

# Add two children for n2
AddEdge graph n2 n0 e0
AddEdge graph n2 n1 e1

# n0 lists children
ListChildren graph n0

# n1 lists children
ListChildren graph n1

# n2 lists children
ListChildren graph n2