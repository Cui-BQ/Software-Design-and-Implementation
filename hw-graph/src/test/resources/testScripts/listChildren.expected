# List children for bides that have 0, 1 and 2 children node.

# Create graphs
created graph graph

# Add 3 node to graph
added node n0 to graph
added node n1 to graph
added node n2 to graph

# Add one child for n1
added edge e2 from n1 to n2 in graph

# Add two children for n2
added edge e0 from n2 to n0 in graph
added edge e1 from n2 to n1 in graph

# n0 lists children
the children of n0 in graph are:

# n1 lists children
the children of n1 in graph are: n2(e2)

# n2 lists children
the children of n2 in graph are: n0(e0) n1(e1)